impeller/renderer/backend/vulkan/capabilities_vk.cc

```diff
@@ -461,6 +461,10 @@ bool CapabilitiesVK::HasExtension(const std::string& ext) const {
   return false;
 }
 
+bool CapabilitiesVK::SupportsPrimitiveRestart() const {
```

Comment:
[You can do the same thing unconditionally for Metal](https://developer.apple.com/documentation/metal/mtlrendercommandencoder/1515699-drawindexedprimitives?language=objc). Metal enables primitive restart by default without any additional configuration. In fact, it don't think it is possible to disable it.

Reply:
This is wired up for metal i this PR.